### PR TITLE
Running sWeights from input args

### PR DESCRIPTION
**Pull request by cpawley**

### CODE REVIEW

**Comment by jackfrost373** [comment]
Looks good to me - perhaps Jonas can comment? Otherwise a merge seems OK

**Comment by cpawley** [comment]
Still needed some bug fixing, it's now ready... @Jonastjep could you review when you have time?

File: analysis/Scripts/dalitzPlane.py [hunk 1]
```diff
@@ -1,84 +1,231 @@
...
-  cuttree = tree.CopyTree(swcuts)
-  print("cutTree nEvents = {0}".format(cuttree.GetEntries()))
-  cuttree.AddFriend("swTree",pwd+"/output/sWeight_swTree.root")
-  weightvar = "swTree.sw_sig"
-else :
-  cuttree = tree
-  weightvar = "1"
 
+#Input dir is where the reduce tuples are, output is where we will make our plots and our friend trees are in sweightdir
+inputdir = "/dcache/bfys/jtjepkem/binned_files/"
```
Comment (Jonastjep) [inline analysis/Scripts/dalitzPlane.py:17]: Should we already make the changes that are compatible with the Imports library that will get merged with the pull #60 ?  This path would be imported as TUPLE_PATH

Comment (Jonastjep) [reply analysis/Scripts/dalitzPlane.py:17]: Other than that -which might just be better left for a future pull request- everything seems perfect!

**Comment by cpawley** [comment]
Yes I agree we should use Imports.Tuple_Path() in the end, just right now that doesn't work because your PR isn't yet merged ;)

**Pull request merged**